intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -209,27 +212,31 @@ \section{Monitoring and managing your job(s)}
 %\shellcmd{qstat $<$jobid$>$}%
 \end{prompt}
 
-To show an estimated start time for you job (note that this may be very inaccurate,
-the margin of error on this figure can be bigger then 100\% due to a sample in a
-population of 1.)
-This command is not available on all systems.
-
-\begin{prompt}
-%\shellcmd{showstart $<$jobid$>$}%
-\end{prompt}
-
-This is only a very rough estimate. Jobs may launch sooner than estimated if
-other jobs end faster than estimated, but may also be delayed if other
-higher-priority jobs enter the system.
-
-To show the status, but also the resources required by the job, with error
-messages that may prevent your job from starting:
-
-\begin{prompt}
-%\shellcmd{checkjob $<$jobid$>$}%
-\end{prompt}
+\ifgent
+%don't display this because it does not work at Gent
```

Comment:
True, but that command - at least in my account - yields:
"The data in the checkjob cache may be outdated. Please contact your admin to look into this."
Also, the result I am getting (from *very* old jobs only, not from a recent/running job) seems like raw database info.

So, I won't mention mycheckjob until (a) it works with up2date info (b) it yields a nitwit-readable output.
Disclaimers:
(1) I am *not* stating that this tool is not useful, nor that your efforts are not appreciated.
(2) The reinstatement of mycheckjob is very very low priority.